trillian/migrillian/core/controller.go

```diff
@@ -84,8 +85,9 @@ func OptionsFromConfig(cfg *configpb.MigrationConfig) Options {
 			EndIndex:      cfg.EndIndex,
 			Continuous:    cfg.IsContinuous,
 		},
-		Submitters:  int(cfg.NumSubmitters),
-		ChannelSize: int(cfg.ChannelSize),
+		Submitters:       int(cfg.NumSubmitters),
+		ChannelSize:      int(cfg.ChannelSize),
+		ConsistencyCheck: !cfg.NoConsistencyCheck,
```

Comment:
Might have been better to have NoConistencyCheck in the struct as well.

Reply:
Done.